guides/common/modules/proc_installing-and-configuring-the-puppet-agent.adoc

```diff
@@ -1,18 +1,21 @@
-[id="Installing_and_Configuring_the_Puppet_Agent_{context}"]
-= Installing and Configuring the Puppet Agent
+[id="Installing_and_Configuring_the_Puppet_Agent_on_Host_Manually_{context}"]
+= Installing and Configuring Puppet Agent on Host Manually
 
-Use this procedure to install and configure the Puppet agent on a host.
-ifndef::orcharhino[]
+Install and configure the Puppet agent on a host manually.
+
+ifndef::managing-configurations-puppet[]
+ifdef::satellite[]
 For more information about Puppet, see {ManagingConfigurationsPuppetDocURL}[Managing Configurations Using Puppet Integration in {Project}].
 endif::[]
+endif::[]
 
 .Prerequisites
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite,orcharhino[]
 * The {Team} {project-client-name} repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
 endif::[]
 ifdef::katello[]
-* Ensure a repository containing the Puppet Agent is enabled on the host, for example https://apt.puppet.com/[apt.puppet.com] or https://yum.puppet.com/[yum.puppet.com].
+* Ensure a repository containing the Puppet agent is enabled on the host, for example https://apt.puppet.com/[apt.puppet.com] or https://yum.puppet.com/[yum.puppet.com].
 endif::[]
```

Comment:
@ekohl Is this condition correct? To my understanding, this is necessary when I DON'T have Katello.

Reply:
If I'm not mistaken, this is for the manual installation procedure. I think it's safe to say that. Even if you have Katello, you should make sure the host has some repo with the Puppet agent available, it can be upstream like you list here or some Katello repo they synced the upstream to. In case of Satellite, Puppet is shipped in Satellite repos, so they need to be subscribed to Katello's (Satellite's) repo that contains it. The 2 main branches are - Foreman/Foreman+Katello/Orcharino (upstream) or Satellite (downstream). With Satellite you must use the content that's shipped with it. For the 3 other possibilities, you need to make sure there's some repo that contains the puppet agent from puppet.com. If you have Katello or Orcharino you can copy and host that puppet agent locally.

Reply:
Indeed what @ares said: you need a repository that contains it. There are a bunch of ways to obtain it and pointing to the official repositories is a good thing in all upstream cases. In the Katello case users can choose to sync the content, which is a good idea but strictly speaking not a hard requirement. I'd at most link to the guide on how to do content syncs and not explain it literally. In Satellite it's preferred to obtain if via Red Hat's repositories.